usr/src/cmd/svc/milestone/net-physical

```diff
@@ -395,6 +395,26 @@ function setup_mtu
     done
 }
 
+#
+# Set up resolvers, if we can
+#
+function conigure_resolv_conf
+{
+    if [[ ${RESOLV_CONF_DONE} == true ]]; then
+        return
+    elif [[ -n ${CONFIG_dns_domain} ]] && [[ -n ${CONFIG_dns_resolvers} ]]; then
+        echo "search ${CONFIG_dns_domain}" > /etc/resolv.conf
+        if [[ -n ${CONFIG_binder_admin_ips} ]]; then
+            for serv in $(echo "${CONFIG_binder_admin_ips}" | sed -e "s/,/ /g"); do
+                echo "nameserver ${serv}" >> /etc/resolv.conf
+            done
+        fi
+        for serv in $(echo "${CONFIG_dns_resolvers}" | sed -e "s/,/ /g"); do
+            echo "nameserver ${serv}" >> /etc/resolv.conf
+        done
+        RESOLV_CONF_DONE=true
+    fi
```

Comment:
It's intentional that we do nothing in this function if RESOLV_CONF_DONE is not true AND both of the two CONFIG_ variables are unset?

Reply:
Yes.

* The previous behavior is that both `CONFIG_` variables must be set.
* Since we explicitly set `RESOLV_CONF_DONE` to `true` at the end of this function, any other value (which should be none) would mean we make an attempt.
* Since it should be impossible for `CONFIG_` values to change during the course of this script, we only need one attempt. Either there's valid values in the config or there isn't.

Reply:
Works for me.